### PR TITLE
Tweed Scrolls

### PR DESCRIPTION
**Pull request by Ralph-Lee-UK**
Corrected formatting issues with EMIPms00469 facsimile and text files. This request seeks to resolve conflicts in a previous pull request yesterday.

### CODE REVIEW

**Comment by Ralph-Lee-UK** [comment]
The conflict in yesterday's pull request is now resolved, following Eugenia's advice.

File: EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
             <ab>
                <pb facs="#facs_1" n="1r"/>
                <cb facs="#facs_1_r1" n="a"/>
-               <lb facs="#facs_1_line_1656341948235_22" n="1" break="no"/>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱ<lb facs="#facs_1_r1l12" n="2" break="no"/>ስ፡ ፩አምላክ፡ ጸሎት፡ በእንተ፡ አው<lb facs="#facs_1_r1l13" n="3" break="no"/>ጸሎት፡ በእንተ፡ አውጽእ፡ አጋንንት፡ ወ<lb facs="#facs_1_r1l14" n="4" break="no"/>ሰይጣናት፡ ዘስሙ፡ ቅዱስ፡ ሚካኤል፡ 
+               <lb facs="#facs_1_line_1656341948235_22" n="1" break="no"/>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱ<lb facs="#facs_1_r1l12" n="2" break="no"/>ሽስAsk Ralph about this፡ ፩አምላክ፡ ጸሎትMy change፡ በእንተ፡ አው<lb facs="#facs_1_r1l13" n="3" break="no"/>ጸሎት፡ በእንተ፡ አውጽእ፡ አጋንንት፡ ወ<lb facs="#facs_1_r1l14" n="4" break="no"/>ሰይጣናት፡ ዘስሙ፡ ቅዱስ፡ ሚካኤል፡
```
Comment (thea-m) [inline EMIP/magicscrolls/EMIPms00469/transkribusTextEMIPms00469.xml:9]: "ሽስAsk Ralph about this፡ ፩አምላክ፡ ጸሎትMy change፡" These seem to be comments maybe not intended for the final version? Also "My change" below

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by Ralph-Lee-UK — COMMENTED** [review]
Indeed, I will review these in further transcriptions if the formatting now works. Thanks for this. @thea-m I think I have done what I need to accept your suggestions!

**Review by Ralph-Lee-UK — COMMENTED** [review]
@thea-m I believe I have removed the comments now

**Review by thea-m — APPROVED** [review]
Thank you!

**Pull request merged**